client/src/pages/course/student/cross-check-review.tsx

```diff
@@ -89,10 +94,13 @@ function Page(props: CoursePageProps) {
       };
     });
 
-    setState({ loading: false, data: solutionReviews ?? [] });
+    const activeSolutionReview = solutionReviews[0];
+
+    form.setFieldValue('comment', activeSolutionReview.comment.slice(markdownLabel.length));
     if (result !== null) {
```

Comment:
It seems that this check here is not necessary, we already checked that `result !== null` on line 72, could you please remove this `if` statement?